### DISCUSSION TITLE
[paste] how to paste content repeatedly ?

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#4487] [Q&A] [answered]
**This is not a question about LazyVim**

In daily use, this problem often causes me inconvenience. After I copy a word or a piece of content, I can't use `p` to paste repeatedly.

If you happen to have solved this problem, I look forward to your answer.

**Comment by dpetka2001** [comment]
For me it just happens naturally. I didn't configure anything to make it happen. Maybe an issue with your terminal? Or another guess maybe issue of the clipboard provider you're using? I'm using `xsel` on Linux Mint 21.3 and latest kitty terminal.

**Reply by iamxiaojianzheng** [reply]
I don't know if we are talking about the same thing?

Suppose I have copied a word Apple, then select banana and press p.

After completing this operation, I can't select other words again and press p to paste and replace them with Apple.

**Reply by dpetka2001** [reply]
Oh yes, that happens because in visual selection the text that you select gets copied in the register after you paste. You probably want to use `P` instead of `p` for that. See `:h v_P` for more info.

**Reply by iamxiaojianzheng** [reply]
Thank you!

**Comment by iamxiaojianzheng** [comment, marked as answer]
```lua
{
  "gbprod/yanky.nvim",
  keys = {
    { "p", "<Plug>(YankyPutBefore)", mode = { "n", "x" }, desc = "Put Text Before Cursor", remap = true },
  },
}
```
